api/wasm/cpp/proxy_wasm_api.h

```diff
@@ -271,10 +261,8 @@ class ContextBase {
   virtual Context* asContext() { return nullptr; }
 
   using HttpCallCallback =
-      std::function<void(std::unique_ptr<WasmData> header_pairs, std::unique_ptr<WasmData> body,
-                         std::unique_ptr<WasmData> trailer_pairs)>;
-  using GrpcSimpleCallCallback =
-      std::function<void(GrpcStatus status, std::unique_ptr<WasmData> message)>;
+      std::function<void(uint32_t, size_t, uint32_t)>; // headers, body_size, trailers
```

Comment:
`headers` / `trailers` - is that a number of items or total length when serialized?

Reply:
number